dandi/cli/cmd_delete.py

```diff
@@ -5,11 +5,16 @@
 
 @click.command()
 @click.option("--skip-missing", is_flag=True)
+@click.option(
+    "--force",
+    is_flag=True,
+    help="Force deletion without requesting interactive confirmation",
+)
 @click.argument("paths", nargs=-1, type=click.Path(exists=False, dir_okay=True))
 @instance_option()
 @devel_debug_option()
 @map_to_click_exceptions
-def delete(paths, skip_missing, dandi_instance, devel_debug=False):
+def delete(paths, skip_missing, dandi_instance, devel_debug=False, force=False):
```

Comment:
```suggestion
def delete(paths, skip_missing, dandi_instance, force, devel_debug=False):
```

Reply:
Because the `--devel-debug` option is only defined when `DANDI_DEVEL` is set, `devel_debug` is not always passed to the command function when it's called, so it needs a default value.  `force`, on the other hand, is always passed to the command function, so giving it a default value is pointless and potentially misleading.

Reply:
Got it - makes sense. Let's do it, thanks!